### ISSUE TITLE
ModuleNotFoundError: No module named 'spsearch.redlist'

### ISSUE THREAD

**Issue by skyre-sky** [AzukiOtter/spsearch#4]
Hello! After installing the module, the following problem appears when I call it, how do I solve it?

Traceback (most recent call last):
  File "E:/- python/07 IUCN API/01 IUCN API.py", line 5, in <module>
    from spsearch.redlist import RedListApiHandler
ModuleNotFoundError: No module named 'spsearch.redlist'

**Comment by aiotter** [comment]
@skyre-sky Sorry for late reply, I missed your msg.
The pip version seems old. You can update it via `pip install -Ue 'git+https://github.com/aiotter/spsearch.git#egg=spsearch'`.

btw, I'm now refactoring this package. The interface around `CodeHierarchySeq` is terrible now. I want to implement it in more easy-to-understand way.
There will be lots of braking changes, so I don't recommend to use it now.

**Comment by stijncon** [comment]
@aiotter Do I understand correctly that you do not recommend using spsearch now for getting data from the EOL structured data API, because you are redoing it? If so, do you have a particular timeframe in mind? I am currently deciding if I should just learn to use the Cypher queries, but wouldn't if your package is easier.

**Comment by aiotter** [comment]
@stijncon You understands correctly. Sorry for saying this, but my priority for this project is not so high. I cannot say when the rewriting is finished.

Actually the reason I made this project is to run my web service, so I'm not so motivated as to cover all the jobs the EOL API can do. While you're getting a general information for a specific species this project may help you, but when you want to get some special information it won't help you: like getting list of the scientific name of the species who pray on the same specific species. You need to learn Cypher to get it.

Currently I have no docs for this project. You may have some trouble learning how to use it.
Cypher is hard to learn at first, but if you want to use the API often, I strongly recommend you to learn Cypher.

---

**TL;DR**
If you want to get information of the species by its scientific name, this project may help you.
If you want to get more than that, go learn Cypher.

**Comment by stijncon** [comment]
@aiotter Thanks for the response, and of course I fully understand! I'll just learn Cypher then.

**Comment by aiotter** [comment]
@stijncon Good luck👍
